llabake/Hello-Books
# Ft logout user 154567758

What does this PR do?
This PR implements log out functionality

Description of Task to be completed?
User should be able to log out from the application

How should this be manually tested?
* clone the repo
* cd into the project directory
* Run npm install to install dependencies
* Run npm run dev to start the client server
* Run npm run start-dev to start the server-side
* Visit the port where the webpack-dev-server started, navigate to the signup page
* Fill the sign in form and click sign in,
* After successful sign in, and navigation across the project, user should be to log out from the application
What are the relevant pivotal tracker stories?
154567758

Any background context you want to add?
N/A

Screenshots
